src/iosrv/init.c

```diff
@@ -232,6 +236,66 @@ dss_topo_init()
 	dss_tgt_nr = dss_tgt_nr_get(dss_core_nr, nr_threads);
 }
 
+/* daos_server dRPC method notifyReady (see also notifyReady) */
+#define SRV_NOTIFY_READY 0
```

Comment:
After discussion around one of Tom's patches, I think we want the method definitions to be enums in src/include/daos/drpc_modules.h, too.

Reply:
Will move.